src/guide/build-and-install.md

```diff
@@ -178,80 +202,107 @@ for details.
+    that’s because the home folder is not a real file system. The fix is to
+    create the `Git` folder.
 
-```bash
-cd ~/Git
-```
+    :::
 
-Then `clone` the Iroha git repository into the folder `~/Git/iroha`.
+2.  Enter the directory you have just created using
 
-```bash
-git clone https://github.com/hyperledger/iroha.git
-```
+    ```bash
+    cd ~/Git
+    ```
 
-This will fetch all of Iroha, including Iroha 1, and the `iroha2-dev`
-branch, which we will touch upon later.
+3.  Then `clone` the Iroha git repository into the folder `~/Git/iroha`:
 
-Change directories
+    ```bash
+    git clone https://github.com/hyperledger/iroha.git
+    ```
 
-```bash
-cd ~/Git/iroha
-```
+    This will fetch all of Iroha, including Iroha 1, and the `iroha2-dev`
+    branch, which we will touch upon later.
 
-and choose the right branch: the main and the latest currently supported
-monthly release of Iroha.
+4.  Change directories:
 
-```bash
-git checkout iroha2
-```
+    ```bash
+    cd ~/Git/iroha
+    ```
 
-After you have successfully cloned the Iroha git repository, and are on the
-correct branch, build the Iroha 2 client using:
+5.  Choose the right branch. You can use the `iroha2-lts` branch, which is
+    the long-term support release, or the branch with the latest stable
+    release (`iroha2-stable`). To checkout the branch with the stable
+    release, run:
 
-```bash
-cargo build -p iroha_client_cli
-```
+    ```bash
+    git checkout iroha2-stable
+    ```
+
+6.  After you have successfully cloned the Iroha git repository and are on
+    the correct branch, build the Iroha 2 client using:
+
+    ```bash
+    cargo build -p iroha_client_cli
+    ```
+
+    ::: info
+
+    We take pride in the fact that Iroha is extremely quick to compile. For
+    reference, compiling hyperledger/substrate takes a good part of ten
+    minutes on a modern M1 machine. Iroha, for comparison, compiles in
+    around one minute.
+
+    :::
+
+## Bring up a minimal network
+
+You can run Iroha
+[directly on bare metal](/guide/advanced/running-iroha-on-bare-metal), but
+we recommend bringing up a network of 4 containerised peers using
+`docker-compose`.
 
 ::: info
 
-We take pride in the fact that Iroha is extremely quick to compile. For
-reference, compiling hyperledger/substrate takes a good part of ten minutes
-to compile on a modern M1 machine. Iroha, for comparison compiles in
-around 1.
+In this tutorial we only cover the default `docker-compose.yml` with the
+image of Iroha 2. You might also be interested in other options for local
+compilation:
 
-:::
+- For testing Iroha code quickly, you can use `docker-compose-single.yml`,
+  which starts a container with a single peer.
+- For testing Iroha code in normal conditions, you can use
+  `docker-compose-local.yml`, which starts 4 connected containers with
+  peers.
+
+Please note that there is an ongoing work to make `docker-compose` even
```

Comment:
We're not making Compose itself customizable, but our container configurations instead: `docker-compose` is an utility that launches groups of containers, while Iroha Swarm will be an utility that makes custom configurations for Docker-compose.

Reply:
rephrased a bit